SQL/Archive/17.1/2017-08-01_remove_reliability_stats.sql

```diff
@@ -0,0 +1,2 @@
+-- Removing reliability statistics as it is more project specific
+DELETE FROM `StatisticsTabs`  WHERE `ID`='4';
```

Comment:
This is bad practice, you are assuming that all projects added their tabs in the same order that your database did. The tab should be removed by reference to its `submoduleName` instead so that it is specific to the reliability tab.